lib/associations/has-many.js

```diff
@@ -333,16 +336,8 @@ class HasMany extends Association {
 
     const oldAssociations = await this.get(sourceInstance, { ...options, scope: false, raw: true });
     const promises = [];
-    const obsoleteAssociations = oldAssociations.filter(old =>
-      !targetInstances.find(obj =>
-        obj[this.target.primaryKeyAttribute] === old[this.target.primaryKeyAttribute]
-      )
-    );
-    const unassociatedObjects = targetInstances.filter(obj =>
-      !oldAssociations.find(old =>
-        obj[this.target.primaryKeyAttribute] === old[this.target.primaryKeyAttribute]
-      )
-    );
+    const obsoleteAssociations = oldAssociations.filter(old => !targetInstances.some(obj => obj[this.target.primaryKeyAttribute] === old[this.target.primaryKeyAttribute]));
+    const unassociatedObjects = targetInstances.filter(obj => !oldAssociations.some(old => obj[this.target.primaryKeyAttribute] === old[this.target.primaryKeyAttribute]));
```

Comment:
What's the max line length?

Reply:
125 (that's how many characters github displays) but I haven't enforced it on JS files for now